emop-core/src/main/java/edu/cornell/emop/util/Util.java

```diff
@@ -47,4 +50,54 @@ public static void replaceFileInJar(String jarPath, String oldPath, String newPa
             ex.printStackTrace();
         }
     }
+
+    /**
+     * Recursive routine accumulating the set of package names within the project.
+     * Borrowed from DSI.
```

Comment:
Add a link to the DSI paper [OR] just remove the "Borrowed from...", since DSI code is not available publicly.